docs/source/getting_started/dataset_creation/dataset_creation_azure_storage.rst

```diff
@@ -1,6 +1,102 @@
 .. _dataset-creation-azure-storage:
 
-Create dataset from azure blob storage
---------------------------------------
+Create a dataset from Azure blob storage
+-----------------------------------------
 
-TODO: Add the tutorial how to create a Lightly dataset from a azure bucket here.
+Lightly allows you to configure a remote datasource like Microsoft's Azure blob storage.
+In this guide, we will show you how to setup your Azure blob storage container and configure your dataset to use said container.
+
+One decision you need to make first is whether you want to use thumbnails.
+Using thumbnails makes the Lightly Platform more responsive, as it's enough to
+load the thumbnails instead of the full images in many cases.
+As a drawback, the thumbnails will be stored in your bucket and thus need storage.
+You have the following three options:
+
+
+- You want to use thumbnails, but don't have them yet. Then you need to give
+  Lightly write access to your bucket to create the thumbnails there for you.
+  The write access can be configured not to allow overwriting and
+  deleting, thus existing data cannot get lost.
+- You already have thumbnails in your bucket with a consistent name scheme, e.g.
+  an image called `img.jpg` has a corresponding thumbnail called `img_thumb.jpg`.
+  In this case, a read access to your bucket is sufficient.
+- You don't want to use thumbnails. Then a read access to your bucket
+  is sufficient. The Lightly Platform will load the full image
+  even when requesting the thumbnail.
+
+Depending on this decision, the following steps will differ slightly.
+
+Setting up Azure
+^^^^^^^^^^^^^^^^^
+
+For the purpose of this guide we assume you have a storage account called `lightlydatalake`.
+We further assume the container you want to use with lightly is called `farm-animals` and already contains images.
+If you don't have a storage account or container yet follow the instructions `here <https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal>`_.
+
+1. In the `Azure Portal <https://portal.azure.com/#home>`_ navigate to `Storage Accounts`.
+2. Select your storage account and go to `Settings > Resource sharing (CORS)`. Allow at least `LIST` and `GET`. For write access also allow `PUT` and `DELETE`.
```

Comment:
Several points here:
- In the newest azure version, there is no "LIST". 
-  I don't understand why you added all 8 methods if only 2 or 4 are need.
- Earlier you tell `The write access can be configured not to allow overwriting and deleting, thus existing data cannot get lost.`. How is this possible here? Doesn't this mean that I don't need DELETE?
- Is is possible to restrict access to only one container? Or will lightly always have access to the full storage account including all of its containers?